db/migrate/20110301154453_add_managed_to_hosts.rb

```diff
@@ -5,7 +5,7 @@ def up
     Host.reset_column_information
 
     for host in Host.unscoped.all
-      host.update_attribute :managed, !!host.operatingsystem_id and !!host.architecture_id and (!!host.ptable_id or !host.disk.empty?)
+      host.update_attribute(:managed, !!host.operatingsystem_id) && !!host.architecture_id && (!!host.ptable_id || !host.disk.empty?)
```

Comment:
I think the brackets are in the correct place due to the operator precedence... but I don't think it's correct logically. I think the intention was to pass the whole expression as the second argument to `update_attribute`.